### ISSUE TITLE
Multiplayer

### ISSUE THREAD

**Issue by artdeell** [dethrace-labs/dethrace#170]
Is it possible to implement multiplayer over tcp/ip or udp?

**Comment by dethrace-labs** [comment]
Yes, definitely! Its on the roadmap, will be easiest to implement it once we have completed the local game mode
